examples/advanced_pytorch/client.py

```diff
@@ -0,0 +1,130 @@
+import utils
+from torch.utils.data import DataLoader
+import torchvision.datasets
+import torch
+import flwr as fl
+import argparse
+from collections import OrderedDict
+import warnings
+
+warnings.filterwarnings("ignore")
+
+
+class CifarClient(fl.client.NumPyClient):
+    def __init__(
+        self,
+        model: torch.nn.Module,
+        trainset: torchvision.datasets,
+        testset: torchvision.datasets,
+        validation_split: int = 0.1,
```

Comment:
Include `device` as parameter.
```suggestion
        device: str = "cpu",
        validation_split: int = 0.1,
```